src/utils/timeFrames.js

```diff
@@ -1,5 +1,10 @@
-export const TimeFrames = {
-  7: 'Soon',
-  14: 'Kind of soon',
-  30: 'Not soon',
+/**
+ * We use the key from these object to add the aria-label
+ * and the color for each item
+ **/
+export const TimeFrameLabels = {
+  inactive: 'This item is inactive due to the amount of time',
+  soon: 'You are expected to buy this item soon',
+  kindOfSoon: 'You are expected to buy this item kind of soon',
+  notAnySoon: 'You are not expected to buy this item any time soon',
 };
```

Comment:
That way it got much better! 🥳

Reply:
Guys it's great, but I think it would be nice to add the number of days

Reply:
Nice object!

However, as Mailliw mentioned I think it could be a good opportunity to create a powerful object (with properties like `numberOfDays`, `label`, `description`, etc.) I believe it could give you more flexibility in your app and reuse some code like the `Add Item`  form.

As always, it's only an idea! 😄